File: Tecnoverde-main/API Gateway/Auth/src/main/java/co/edu/javeriana/ms/LoadBalancer/entities/Cliente.java
package co.edu.javeriana.ms.LoadBalancer.entities;

import jakarta.persistence.Basic;
import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.Id;
import jakarta.persistence.ManyToOne;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@Entity
public class Cliente {

    @Id
    @GeneratedValue
    private Long id;
    @Basic
    private String username;
    @Basic
    private String password;
    @Basic
    private String nombre;
    @Basic
    private String edad;
    @Basic
    private String descripcion;
    @ManyToOne
    private Foto foto;

}

File: Client/llamadas/cuentas/tipos/cliente.go
package tipos;

type Cliente struct {
	Username    string `json:"username"`
	Nombre      string `json:"nombre"`
	Edad        string `json:"edad"`
	Descripcion string `json:"descripcion"`
	Foto        string `json:"foto"`
}

func (c Cliente) ObtenerColumnas() []string {
	return []string{
		"Username",
		"Nombre",
		"Edad",
		"Descripcion",
		"Foto",
	}
}

func (c Cliente) ObtenerFilas() [][]string {
	return [][]string{
		{c.Username, c.Nombre, c.Edad, c.Descripcion, c.Foto},
	}
}